src/main/java/frc/robot/subsystems/shooter/ShooterSS.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.subsystems.shooter;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.components.TrigonTalonFX;
+import frc.robot.constants.RobotConstants.ShooterConstants;
+import frc.robot.subsystems.TestableSubsystem;
+import frc.robot.utilities.EncoderConversions;
+
+public class ShooterSS extends SubsystemBase implements TestableSubsystem {
+    private final TrigonTalonFX masterMotor;
+
+    public ShooterSS() {
```

Comment:
should the hood be apart of this?

Reply:
No. Separate especially since on how we doing the shooting